src/components/home/OurManufacturingServices.jsx

```diff
@@ -1,7 +1,57 @@
 import React from "react";
+import Image from "next/image";
+import ServiceDescription from "@/components/home/ServiceDescription";
+import ClampsAndBracket from "../../../public/images/home/ClampsAndBracket.webp";
+import WireHarnesses from "../../../public/images/home/WireHarnesses.webp";
 
 const OurManufacturingServices = () => {
-  return <div>OurManufacturingServices</div>;
+  return (
+    <div className="w-full bg-jm-blue-200">
+      <div className="flex items-center justify-center">
+        <Image
+          src="/svg/GearIcon.svg"
+          alt="Gear Icon"
+          className="relative left-6 top-1"
+          style={{ zIndex: 1 }}
+          width={32}
```

Comment:
don't need it, once you import images at the top. width and height can be tailwind utility classes